var/spack/repos/builtin/packages/py-cheetah/package.py

```diff
@@ -0,0 +1,37 @@
+##############################################################################
+# Copyright (c) 2013-2018, Lawrence Livermore National Security, LLC.
+# Produced at the Lawrence Livermore National Laboratory.
+#
+# This file is part of Spack.
+# Created by Todd Gamblin, tgamblin@llnl.gov, All rights reserved.
+# LLNL-CODE-647188
+#
+# For details, see https://github.com/spack/spack
+# Please also see the NOTICE and LICENSE files for our notice and the LGPL.
+#
+# This program is free software; you can redistribute it and/or modify
+# it under the terms of the GNU Lesser General Public License (as
+# published by the Free Software Foundation) version 2.1, February 1999.
+#
+# This program is distributed in the hope that it will be useful, but
+# WITHOUT ANY WARRANTY; without even the IMPLIED WARRANTY OF
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the terms and
+# conditions of the GNU Lesser General Public License for more details.
+#
+# You should have received a copy of the GNU Lesser General Public
+# License along with this program; if not, write to the Free Software
+# Foundation, Inc., 59 Temple Place, Suite 330, Boston, MA 02111-1307 USA
+##############################################################################
+from spack import *
+
+
+class PyCheetah(PythonPackage):
+    """Cheetah is a template engine and code generation tool."""
+
+    homepage = "https://pypi.python.org/pypi/Cheetah/2.4.4"
+    url      = """\
+https://pypi.python.org/packages/b4/18\
```

Comment:
@mmpozulp If I try to visit `https://files.pythonhosted.org/packages/b4/18a9b2f0f09e99024f4a9ea1ab80b6807aaecb7d1552529ea3590ad3d7bc93/Cheetah-2.3.0.tar.gz` I'm getting a "file not found error", perhaps this has updated since the PR was opened?

Reply:
Try:
https://pypi.io/packages/source/C/Cheetah/Cheetah-2.3.0.tar.gz

Reply:
Ah and that URL format would be better for Spack's version substitution as well (e.g. if more versions of py-cheetah were added)

Reply:
I used @adamjstewart's url and made it a single line. Thanks for reminder about flake8 skipping URLs.